### PR TITLE
Fix hours update.sh

### PR DESCRIPTION
**Pull request by gijskoning**

### CODE REVIEW

File: update.sh [hunk 1]
```diff
@@ -1,9 +1,9 @@
 #!/bin/bash -e
 
 cd "$(dirname "$0")"
-HH="[0-3][0-9]"
+HH="[0-1][0-8]" # Ultra-Rapid releases with file suffixes 00,06,12,18
```
Comment (gregjhogan) [inline update.sh]: if it isn't for matching all valid hours I probably wouldn't call it `HH` (the goal of these variables was re-usability), but maybe it is fine to match all hour possibilities?

Comment (gijskoning) [reply update.sh]: I changed it to use all hours and kept the HH name

File: update.sh [hunk 2]
```diff
@@ -1,9 +1,9 @@
 #!/bin/bash -e
 
 cd "$(dirname "$0")"
-HH="[0-3][0-9]"
+HH="[0-2][0-9]"
 DD="[0-3][0-9]"
-MM="[0-1][0-9]"
+MM="[0-3][0-9]"
```
Comment (gregjhogan) [inline update.sh]: so what is this if it is not the month as 00-11 or 01-12?

Comment (gijskoning) [reply update.sh]: This is me being not so smart. I wasn't thinking. Thanks for noticing

**Pull request merged**